cmd/peer-s3-server.go

```diff
@@ -80,13 +80,19 @@ func (s *peerS3Server) HealthHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func listBucketsLocal(ctx context.Context, opts BucketOptions) (buckets []BucketInfo, err error) {
-	quorum := (len(globalLocalDrives) / 2)
+	z, ok := newObjectLayerFn().(*erasureServerPools)
+	if !ok {
```

Comment:
If this is okay, then it means the object storage is initalized and 'z' is not nil ?

maybe we should return errServerNotInitialized in that case